composer.json

```diff
@@ -40,7 +40,8 @@
         "zendframework/zend-hydrator":       "^1.1",
         "zendframework/zend-mvc":            "^2.5.2",
         "zendframework/zend-servicemanager": "^2.7.5 || ^3.0.3",
-        "symfony/console":                   "~2.5|~3.0"
+        "symfony/console":                   "~2.5|~3.0",
+        "onurb/doctrine-metadata-grapher":   "~1.0"
```

Comment:
I like the idea of getting rid of one piece of maintenance here, and to stash it away, but will you keep up with the maintenance?

I'm not questioning your technical skills, I'm just asking how cautious you will be with upgrades and changes (semver-wise).